src/ripple/rpc/impl/ServerHandlerImp.h

```diff
@@ -167,13 +166,21 @@ class ServerHandlerImp
         std::shared_ptr<JobCoro> jobCoro,
         std::string forwardedFor, std::string user);
 
-private:
     bool
-    isWebsocketUpgrade (http_request_type const& request);
+    isWebsocketUpgrade(http_request_type const& request) const;
```

Comment:
`isWebsocketUpgrade` `isStatusRequest` `unauthorizedResponse` and `authorized` do not need to be members. They can be replaced with `static` functions in the cpp file. Here's a commit to cherry pick if you want: https://github.com/seelabs/rippled/commit/fcd2a7200a4e7fdcb2ff1876d27d9cbbebc17888